app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -335,6 +336,10 @@ class DiscoveryView extends React.Component {
     });
   };
 
+  handleSampleSelected = ({ sample, event }) => {
```

Comment:
Please base this on my current open PR. Already have a function like this defined (triggered by the search box).

Reply:
whoops sorry